EventFilter/CTPPSRawToDigi/interface/RawDataUnpacker.h

```diff
@@ -53,6 +53,9 @@ namespace ctpps
 
       /// Process one Opto-Rx frame in parallel (new) format
       int ProcessOptoRxFrameParallel(const word *buffer, unsigned int frameSize, TotemFEDInfo &fedInfo, SimpleVFATFrameCollection *fc) const;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.